src/zhinst/toolkit/nodetree/nodetree.py

```diff
@@ -121,6 +121,21 @@ def add(self, node: t.Union[Node, str], value: t.Any) -> None:
         except AttributeError as exception:
             raise AttributeError("No set transaction is in progress.") from exception
 
+    def add_raw_list(self, node_value_pairs: t.List[t.Tuple[str, t.Any]]):
```

Comment:
```suggestion
    def add_raw_list(self, node_value_pairs: t.List[t.Tuple[str, t.Any]]) -> None:
```
missing return statement